comp/core/tagger/impl/local_tagger.go

```diff
@@ -35,8 +39,17 @@ const (
 	externalDataCacheTTL = 1 * time.Second
 )
 
-// Tagger is the entry class for entity tagging. It hold the tagger collector,
-// memory store, and handles the query logic. One should use the package
+// datadogConfig contains the Agent configuration.
```

Comment:
The diff on this file is a bit large, so to make sure I'm not missing anything, did you change something or just copied the logic that we had in `impl/tagger.go` ?

Reply:
No it's just a copy.